docs/UserGuide.md

```diff
@@ -191,7 +191,7 @@ What you will see:
 
 |                                                  |
 |:------------------------------------------------:|
-| ![AddCommandResult](images/AddCommandResult.png) ||
+| ![AddCommandResult](images/AddCommandResult.png) |
 
 
 <div markdown="block" class="alert alert-success">
```

Comment:
is this what you intended? just checking
![Screenshot 2022-11-06 at 11 10 15 PM](https://user-images.githubusercontent.com/77665182/200178796-a6118dcd-1e4b-4c73-869f-2b5433aaa097.png)

Reply:
yup, otherwise it appears as 2 columns and the one on the right will have no contents

Reply:
oh but what about the empty header? based on the other tables I'm guessing you wanted to put "What you will see" inside as the header?

Reply:
![Screenshot 2022-11-07 at 12 02 47 AM](https://user-images.githubusercontent.com/77665182/200181305-36a0c2d3-8995-4990-92ea-2c7ab1e285fb.png)

Reply:
resolved